lib/omnibus/packagers/msi.rb

```diff
@@ -96,32 +74,11 @@ class Packager::MSI < Packager::Base
 
         # This assumes, rightly or wrongly, that any installers we want to bundle
         # into our installer will be downloaded by omnibus and put in the cache dir
-
         if bundle_msi
-          shellout! <<-EOH.split.join(' ').squeeze(' ').strip
-          candle.exe
-            -nologo
-            #{wix_candle_flags}
-            -ext WixBalExtension
-            #{wix_extension_switches(wix_candle_extensions)}
-            -dOmnibusCacheDir="#{windows_safe_path(File.expand_path(Config.cache_dir))}"
-            "#{windows_safe_path(staging_dir, 'bundle.wxs')}"
-          EOH
+          shellout!(candle_command(true))
```

Comment:
It'd be nice to be able to use keyword arguments here so we know what true means without having to go look for the method. I've submitted #567 to bump the minimum ruby version for omnibus to 2.0 since we're using 2.1.5 anyway.

```
shellout!(candle_command(is_bundle: true))

def candle_command(is_bundle: false)
  # blah
end
```